pkg/io/opener.go

```diff
@@ -81,6 +82,7 @@ func LogClose(c io.Closer) {
 }
 
 func (o opener) openGCS(path string) (*storage.ObjectHandle, error) {
+	path = strings.TrimPrefix(path, "file://")
```

Comment:
Seems like we should only do this when its prefix isn't `gs://`, right?

Reply:
I did it like this because TrimPrefix has the following behaviour:
```
// TrimPrefix returns s without the provided leading prefix string.
// If s doesn't start with prefix, s is returned unchanged.
```
But I guess it would be more obvious if I write it like this:

```
if strings.HasPrefix(path, "file://") {
   path = strings.TrimPrefix(path, "file://")
}
```

Reply:
The edge case here is that is `file://gs://foo` should be treated as a local path, but will currently go to GCS

Reply:
Okay I moved it. So the prefix is only stripped if the gs:// prefix is not detected. But I doubt the `os.Create` function can handle paths like `gs://foo` (It couldn't even handle `f, err := os.Create("file:///tmp/test")`